docs/release_process.rst

```diff
@@ -0,0 +1,165 @@
+
+.. note::
+
+  The changelog creation is unlikely to be accurate when creating a bug fix/hot fix release on a previously released official version.
+  At this time, that means some manual editing of the changelog will need to happen when creating a bug fix/hot fix release.
+
+.. figure:: ./images/release_process/prepare_release_builder_interface.png
+   :align: center
+   :alt: Buildbot prepare release interface.
+   :name: libcellml_release_process_prepare_release_builder_interface
+
+   Prepare release interface.
+
+There are no options for the prepare release builder, there is only one place you can prepare a release from, there is only one place a release is going to be created.
+The only thing you can do is start a build, (:numref:`libcellml_release_process_prepare_release_builder_interface`).
+
+The prepare release builder will kick off a round of unit tests as part of the preparation process.
+
+When the release has been prepared the *release_staging_<version number>* branch will have been created and a changelog and table of contents entries for the changelog will have been created.
+The changes made for the new changelog should be the only changes from the current *main* branch.
+
+Step 3 - Creating the release
+=============================
+
+A release is created using the *Create Release Builder* (:numref:`libcellml_release_process_create_release`).
+
+.. figure:: ./images/release_process/create_release_builder.png
+   :align: center
+   :alt: Buildbot create release builder interface.
+   :name: libcellml_release_process_create_release
+
+   Create release builder on Buildbot.
+
+A release can only be created from a *release_staging_<version number>* branch.
+The create release builder interface interrogates `cellml/libcellml <https://github.com/cellml/libcellml>`__ for potential release branches.
+Select the release preparation branch (there should only ever be one), to create the release from, and start the build with the *Start Build* button, :numref:`libcellml_release_process_create_release_builder_interface`.
+
+.. figure:: ./images/release_process/create_release_builder_interface.png
+   :align: center
+   :alt: Buildbot create release interface.
+   :name: libcellml_release_process_create_release_builder_interface
+
+   Create release interface showing the branch a release can be created from.
+
+The create release build creates a release on GitHub and adds tags to identify where the release was created from.
+The current status for the *release_staging_<version number>* branch will be applied to the *release* branch.
+The tagged *release* branch is where the GitHub release will be created from.
+When the release is created GitHub actions will take over to build all the assets.
+The assets here are the binaries (installers, archives) from: Windows; macOS; Ubuntu; the Python wheels; and, the Javascript package.
```

Comment:
"...( installers and archives)..."
"... from: Windows, Ubuntu, macOS, Python wheels, and Javascript package."